backend/src/main/java/develup/submission/SubmissionResponse.java

```diff
@@ -0,0 +1,20 @@
+package develup.submission;
+
+public record SubmissionResponse(
+        Long id,
+        Long memberId,
+        Long missionId,
+        String url,
+        String comment
+) {
+
+    public static SubmissionResponse of(Submission submission) {
+        return new SubmissionResponse(
+                submission.getId(),
+                submission.getMember().getId(),
+                submission.getMission().getId(),
```

Comment:
[Approve]

getter 연속 호출은 가독성을 낮춘다고 생각했어요.
아래와 같은 느낌은 어떨까요? 리뷰의 의견을 자유롭게 공유해주세요.

```suggestion
                submission.getMemberId(),
                submission.getMissionId(),
```

Reply:
전 둘 다 상관 없다파여서 아톰의 의견 반영했습니다! 😆